### PR TITLE
Set git hash in ampIntegralMatrixMetadata

### PR DESCRIPTION
**Pull request by suhlatwork**
Set the hash to a value if all amplitude files have the same hash, and this hash is the same as the current ROOTPWA hash. Otherwise leave it empty (as is currently the case).

### CODE REVIEW

**Comment by suhlatwork** [comment]
Now always set the git hash to the revision of the `calcIntegral` executable.

File: pyInterface/module/_integrals.py [hunk 1]
```diff
@@ -8,8 +8,12 @@ def calcIntegrals(integralFileName, eventAndAmpFileDict, multiBin, weightFileNam
 		pyRootPwa.utils.printWarn("cannot open output file '" + integralFileName + "'. Aborting...")
 		return False
 	integralMetaData = pyRootPwa.core.ampIntegralMatrixMetadata()
+	integralMetaData.setGitHash(pyRootPwa.core.gitHash())
 	integralMetaData.setBinningMap(multiBin.boundaries)
 	integrals = []
+	# initialize the git hash with the hash of the current ROOTPWA. this
+	# is compared to the ones from the amplitude files and only kept if all
+	# agree. otherwise an empty string will be set.
 	for eventFileName, ampFileDict in eventAndAmpFileDict.iteritems():
```
Comment (bgrube) [inline pyInterface/module/_integrals.py]: The above comments need to be removes

**Comment by suhlatwork** [comment]
Removed the outdated comment, thank you for noticing.

**Comment by bgrube** [comment]
Merged. Thank you!

**Pull request merged**